ubuntu-clock-app/snapcraft.yaml

```diff
@@ -39,10 +40,13 @@ parts:
       - qtubuntu-desktop
       - qml-module-qtsysteminfo
       - qtmir-desktop
+      - qtorganizer5-eds
       - mir-graphics-drivers-desktop
     snap:
       - -usr/share/doc
       - -usr/include
+    after:
+      - desktop-gtk3
```

Comment:
We need glib/gsettings paths and some environment initializations that is done by this launcher. This is necessary to communicate with EDS.